components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -1,4 +1,20 @@
+// If unset, set `process.env.NTBA_FIX_319` to true to enable cancellation of
```

Comment:
Out of curiosity are we also migrating this to ES6 modules?

Reply:
I converted the JS files in `telegram_bot_api` from CommonJS to ESModules.